### PR TITLE
use Status.DockerImageRepository instead of DockerImageRepository

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by mfojtik** [comment]
lgtm

**Comment by bparees** [comment]
@smarterclayton @ironcladlou this should be all that's needed, right?

**Comment by bparees** [comment]
[test]

File: pkg/build/controller/image_change_controller.go [hunk 1]
```diff
@@ -66,7 +66,7 @@ func (c *ImageChangeController) HandleImageRepo() {
 
 			// (must be different) to trigger a build
 			if icTrigger.LastTriggeredImageID != imageID {
-				imageSubstitutions[icTrigger.Image] = imageRepo.DockerImageRepository + ":" + imageID
+				imageSubstitutions[icTrigger.Image] = imageRepo.Status.DockerImageRepository + ":" + imageID
```
Comment (smarterclayton) [inline pkg/build/controller/image_change_controller.go]: I think if Status.DockerImageRepository is empty we can't even trigger.

Comment (smarterclayton) [inline pkg/build/controller/image_change_controller.go]: That check probably needs to be above.

Comment (bparees) [inline pkg/build/controller/image_change_controller.go]: @smarterclayton  :  added

**Comment by smarterclayton** [comment]
LGTM [merge]

**Pull request merged**